### ISSUE TITLE
Add a better descripition above the statement language dropdown.

### ISSUE THREAD

**Issue by EdOverflow** [liberapay/liberapay.com#934]
I only just noticed that one can store statements in various languages. In my opinion, this could be made clearer by adding a short description above the dropdown in the settings panel.

![image](https://user-images.githubusercontent.com/18099289/35038689-d861e37c-fb7b-11e7-8f3c-bf4c14b2c5d1.png)

**Comment by Changaco** [comment]
We already have "Liberapay allows you to internationalize your texts. Use the selector below to switch between languages." above.

**Comment by EdOverflow** [comment]
Yep, I saw that, but I would prefer that the description be placed above the dropdown instead.
